### PR TITLE
Changes dark grey text to a lighter color for better readability

### PR DESCRIPTION
**Pull request by LauraNK**
The dark grey text was changed to a light grey or white, which is much more readable against the dark blue background.

Original: http://i.imgur.com/wwzalF4.png
With fix: http://i.imgur.com/r7ptFOZ.png

Mostly impacts the Moments capsule pages. Also changes the bio descriptions when an account is shown in a search result, and a line of text in the "change" trends window.
Fix for #17.

### CODE REVIEW

File: sass/_twitter-night-mode.scss [hunk 1]
```diff
@@ -194,6 +194,10 @@ body.three-col.logged-in {
 	}
 }
 
+.ProfileCard-bio, .ProfileCard-locationAndUrl {
+	color: $text-grey!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss:198]: Not recomend use important rule

File: sass/_twitter-night-mode.scss [hunk 2]
```diff
@@ -29,7 +29,7 @@ body.three-col.logged-in {
 	}
 
 	.QuoteTweet-fullname {
-		color: $quote-tweet-fullname-color;
+		color: $tweet-name-color!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss]: Not recomend use important rule

File: sass/_twitter-night-mode.scss [hunk 3]
```diff
@@ -437,6 +441,24 @@ body.three-col.logged-out {
 	padding: 3px;
 	color: lighten($text-grey, 7%)!important;
 }
-.twitter-atreply, .js-display-url{
+.twitter-atreply, .js-display-url, .twitter-hashtag {
+	color: $text-grey!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss:445]: Not recomend use important rule

File: sass/_twitter-night-mode.scss [hunk 4]
```diff
@@ -437,6 +441,24 @@ body.three-col.logged-out {
...
+}
+
+.MomentFormatLink-title, .MomentFormatLink-description {
+	color: $text-grey!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss:449]: Not recomend use important rule

File: sass/_twitter-night-mode.scss [hunk 5]
```diff
@@ -437,6 +441,24 @@ body.three-col.logged-out {
...
+}
+
+.QuoteTweet-fullname.u-linkComplex-target {
+	color: $white!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss]: Not recomend use important rule

File: sass/_twitter-night-mode.scss [hunk 6]
```diff
@@ -437,6 +441,24 @@ body.three-col.logged-out {
...
+
+.QuoteTweet-text .pretty-link b, .QuoteTweet-text .pretty-link s {
 	color: $text-grey!important;
-}
\ No newline at end of file
+}
+
+// Change trends window
+.js-location-nonpersonalized {
+	color: $text-grey!important;
```
Comment (alxlark) [inline sass/_twitter-night-mode.scss:462]: Not recomend use important rule

Comment (tsriram) [reply sass/_twitter-night-mode.scss:462]: @alxlark thanks for the review. Yeah, it's not recommended to use `!important`, but it's required in some cases. Not sure about these ones. I've myself used `!important` in many places and hope I find time to clean them up.

@LauraNK see if these work without `!important` and remove them. Also, better not use classes that start with `js-` as they're meant to be used by JavaScript and not for styling. See if the same element uses a different class for styles and override them.

**Comment by LauraNK** [comment]
Changes made :)

**Comment by tsriram** [comment]
Thanks @LauraNK :)

**Comment by tsriram** [comment]
I'm reverting some of these styles and changing the link color to blue to be in line with color scheme used in dark mode in twitter's mobile apps.

**Pull request merged**